### PR TITLE
Tidy up and add missing coverage

### PR DESCRIPTION
**Pull request by stewartie4**
So far:
- missing param check on cancel allocation
- missing param check on finalize allocation
- update blobber terms param
- upload large file
- upload file larger than size of allocation
- upload file larger than (allocation size/ (1 + num parity shards))
- share entire directory
- download entire directory
- download entire shared directory  
- zwallet getid
- zwallet lockconfig

Additional tidy up:
- Fixed bugs in finalize test 
- extract cancel and finalize tests to own file

### CODE REVIEW

File: tests/cli_tests/zboxcli_cancel_allocation_test.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+func TestCancelAllocation(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Cancel allocation immediately should work", func(t *testing.T) {
+		t.Parallel()
+
+		allocationID := setupAllocation(t, configPath)
+
+		output, err := cancelAllocation(t, configPath, allocationID, false)
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_cancel_allocation_test.go:27]: Should we keep retry true here?

File: tests/cli_tests/zboxcli_download_test.go [hunk 2]
```diff
@@ -137,6 +137,140 @@ func TestDownload(t *testing.T) {
...
+			"authticket": authTicket,
+			"localpath":  "tmp/dir",
+			"remotepath": "/" + filename,
+		}), false)
+		require.NotNil(t, err, strings.Join(output, "\n"))
+		require.Len(t, output, 1)
+		require.Equal(t, "Error in file operation: No minimum consensus for file meta data of file", output[0])
+	})
+
+	t.Run("Download Entire Shared Folder Should Fail", func(t *testing.T) {
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_download_test.go:222]: This should fail? Sounds like it should pass and might be a fixme?

**Review by Kishan-Dhakan — APPROVED** [review]
Great coverage! Minor queries but LGTM otherwise

File: tests/cli_tests/zboxcli_download_test.go (hunk 2 above)
Comment (stewartie4) [reply tests/cli_tests/zboxcli_download_test.go:222]: I get your point but "please get files from folder, and download them one by one" suggests it's expected 
but I'll raise it to confirm

**Pull request merged**